src/test/java/greencity/service/impl/OpenHoursServiceImplTest.java

```diff
@@ -149,4 +145,35 @@ public void findAllTest() {
 
         assertEquals(genericEntities, foundEntities);
     }
+
+    @Test
+    public void getOpenHoursByPlaceTest() {
+        Place place = Place.builder().build();
```

Comment:
What about keyword "new"?

Reply:
I think that there is no difference between creating an object using builder and constructor in such case but I refixed this to "new".